app/containers/FileModal.js

```diff
@@ -75,19 +98,31 @@ const ModalContent = React.memo(({
 		const uri = formatAttachmentUrl(attachment.video_url, user.id, user.token, baseUrl);
 		return (
 			<SafeAreaView style={styles.safeArea}>
-				<VideoPlayer
+				<Video
 					source={{ uri }}
-					onBack={onClose}
-					disableVolume
+					rate={1.0}
+					volume={1.0}
+					isMuted={false}
+					resizeMode='cover'
+					shouldPlay
+					isLooping={false}
+					style={styles.video}
+					posterStyle={styles.poster}
+					usePoster
```

Comment:
What `poster` means? Thumb?
If so, it appears not to be working.

Reply:
no, poster is an image that is displayed before video has loaded. I removed it because in fact it looks better without it.